next.config.js

```diff
@@ -11,21 +11,12 @@ const nextConfig = {
         headers: [
           {
             key: 'X-Frame-Options',
-            value: 'DENY'
-          }
+            value: 'DENY',
+          },
         ],
-      },      
+      },
     ]
   },
-  async redirects() {
-    return [
-      {
-        source: "/",
```

Comment:
This should work on Netlify but for some reason it isn't. We do similar redirects in the app. We'll revisit this after launch day.